core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -130,22 +130,26 @@ public KafkaSchemaRegistry(SchemaRegistryConfig config,
     this.schemaRegistryZkNamespace = config.getString(SchemaRegistryConfig.SCHEMAREGISTRY_ZK_NAMESPACE);
     this.isEligibleForMasterElector = config.getBoolean(SchemaRegistryConfig.MASTER_ELIGIBILITY);
     this.myIdentity = new SchemaRegistryIdentity(host, port, isEligibleForMasterElector);
-    this.kafkaClusterZkUrl =
-        config.getString(SchemaRegistryConfig.KAFKASTORE_CONNECTION_URL_CONFIG);
-    this.zkSessionTimeoutMs =
-        config.getInt(SchemaRegistryConfig.KAFKASTORE_ZK_SESSION_TIMEOUT_MS_CONFIG);
-    this.kafkaStoreTimeoutMs = 
-        config.getInt(SchemaRegistryConfig.KAFKASTORE_TIMEOUT_CONFIG);
     this.serializer = serializer;
     this.defaultCompatibilityLevel = config.compatibilityType();
     this.guidToSchemaKey = new HashMap<Integer, SchemaKey>();
     this.schemaHashToGuid = new HashMap<MD5, SchemaIdAndSubjects>();
+    this.zkAclsEnabled = checkZkAclConfig(config);
+
+    KafkaStoreConfig kafkaStoreConfig = new KafkaStoreConfig(config.originalsWithPrefix(KafkaStoreConfig.KAFKASTORE_CONFIG_PREFIX));
```

Comment:
This isn't quite enough because you have at least the `groupid` that is now required for the KafkaStore but will not be included in the `SchemaRegistryConfig`. This ties back to the `test-group` issue -- this is where you would need to get the identity info that got dropped from `KafkaStore`.

Easy way to test this -- try to run schema registry with a simple default config :) Currently it will crash immediately because of this. (Generally, a good sanity check when making substantial changes like this is to just execute the entire quickstart.)